meraki.py

```diff
@@ -3680,3 +3680,27 @@ def getmxperf(apikey, networkid, serial, suppressprint=False):
     dashboard = requests.get(geturl, headers=headers)
     result = __returnhandler(dashboard.status_code, dashboard.text, calltype, suppressprint)
     return result
+
+def get_video_link(apikey, serial, timestamp=None, suppressprint=False):
```

Comment:
The function was named according to PEP-8, which prefers words separated by underscores. It does look a bit different when compared to the other functions throughout the rest of the module, however I think this is far easier to read. Should we keep it as it is, or change it to something like getvideolink() to match the rest of the module?